domino/lib/src/experimental/idom_browser.dart

```diff
@@ -100,13 +100,11 @@ class BrowserDomContext implements DomContext<Element, Event> {
 
     // fallback: create new Element
     final newElem = Element.tag(tag);
-    if (key != null || onCreate != null || onRemove != null) {
```

Comment:
Why remove this? If the DOM element is simple, there is not reason create and attach an extra object to it.

Reply:
Event handlers are stored in the extra object, so the extra has to be created for every element. But the creation can e moved to the first event() call.

Reply:
Yeah, but this is only required to be created conditionally, e.g. when there is an event handler. Let's not create it for every Element.